venues/ML_Reproducibility_Challenge/2020/process/claimProcess.py

```diff
@@ -1,11 +1,38 @@
 def process_update(client, note, invitation, existing_note):
     CONFERENCE_ID = 'ML_Reproducibility_Challenge/2020'
+    SHORT_PHRASE = "ML Reproducibility Challenge 2020"
+    NOTIFICATION_SUBSCRIPTION_ID = CONFERENCE_ID + '/-/Notification_Subscription'
 
     action = 'posted'
     if existing_note:
         action = 'deleted' if note.ddate else 'updated'
 
     if action == 'posted':
+
+        # get all notification tags for this paper
+        notifications = client.get_tags(forum=note.forum, invitation=NOTIFICATION_SUBSCRIPTION_ID)
+
+        email_list = [tag.signatures[0] for tag in notifications if tag.tag == 'Subscribe']
+        all_notifiers = [tag.signatures[0] for tag in notifications]
+
+        # get submission author and add to email list if author hasn't set a notification tag
+        forumNote = client.get_note(id=note.forum)
+
+        # add comment author to email list if does not have a tag
+        if note.signatures[0] and note.signatures[0] not in all_notifiers:
```

Comment:
it is not necessary to do this because you are not sending an email to the user that created the claim